aws_marketplace/curating_aws_marketplace_listing_and_sample_notebook/Algorithm/Sample_Notebook_Template/title_of_your_product-Algorithm.ipynb

```diff
@@ -0,0 +1,867 @@
+    "    \n",
+    "    \n",
+    "You are also recommended to provide notes on how can the customer scale usage of your algorithm for really large datasets. \n",
+    "    \n",
+    "**You are strongly recommended to provide this section with tuning guidelines and code for running an automatic tuning job**.\n",
+    "    \n",
+    "For information about Automatic model tuning, see [Perform Automatic Model Tuning](https://docs.aws.amazon.com/sagemaker/latest/dg/automatic-model-tuning.html)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### A. Tuning Guidelines"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<font color='red'>For Seller to update: Provide guidelines on how customer can tune their ML model effectively using your algorithm in following cell. Provide details such as which parameter can be tuned for best results.  </font>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": []
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### B. Define Tuning configuration"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<font color='red'>For seller to update: Provide a recommended hyperparameter range configuration in the following cell. This configuration would be used for running an [HPO](https://docs.aws.amazon.com/sagemaker/latest/dg/automatic-model-tuning-how-it-works.html) job. For More information, see [Define Hyperparameters](https://docs.aws.amazon.com/sagemaker/latest/dg/automatic-model-tuning-define-ranges.html)</font>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "hyperparameter_ranges = {}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<font color='red'>For seller to update: As part of your algorithm, please provide multiple objective metrics so that customer can choose a metric for tuning \n",
+    "a custom ML model. Update the following variable with a most suitable/popular metric that your algorithm emits. For more information, see [Define Metrics](https://docs.aws.amazon.com/sagemaker/latest/dg/automatic-model-tuning-define-metrics.html) </font>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "objective_metric_name = '<For Seller to specify : Provide an appropriate objective metric emitted by the algorithm>'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<font color='red'>For seller to update: Specify whether to maximize or minimize the objective metric, in following cell.</font>"
```

Comment:
I'd swap all of these out and make them consistent, and not switch btwn "specify" and "update". So the customer/seller can just search for "For seller to update:" and they know they got them all.

Reply:
i incorporated the same for all except one which says 'For seller to read' since there is nothing to update.